### PR TITLE
Fix battle cards

### PR DESCRIPTION
**Pull request by ridhoq**
Battling cards got broke when we stopped importing cards into `interesting_cards` (6b0de7c0d11b8dd76dcacc7e87695bad845a3980)

Moved all testing cards into a module and import as necessary

### CODE REVIEW

File: kaori/plugins/gacha/engine/test/card_simulator.py [hunk 1]
```diff
@@ -3,9 +3,7 @@
 import arrow
 import mdv
 
-from . import hp_cards, dmg_cards, balanced_cards, meme_cards
-from .dmg_cards import dmg_no_invest
-from .hp_cards import hp_no_invest
+from .cards import *
```
Comment (austinpray) [inline kaori/plugins/gacha/engine/test/card_simulator.py:6]: :+1:

File: kaori/plugins/gacha/engine/test/cards/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from .balanced_cards import *
```
Comment (austinpray) [inline kaori/plugins/gacha/engine/test/cards/__init__.py:1]: module is much better :+1:

File: kaori/plugins/gacha/engine/test/card_simulator.py [hunk 3]
```diff
@@ -3,9 +3,7 @@
 import arrow
 import mdv
 
-from . import hp_cards, dmg_cards, balanced_cards, meme_cards
```
Comment (austinpray) [inline kaori/plugins/gacha/engine/test/card_simulator.py:6]: Damn so I expected this to break the card collection below

DeepinScreenshot_select-area_20200504093616

But looks like `from .cards import *` imports both what is in __init__.py and all the modules that are called in there?

Comment (ridhoq) [reply kaori/plugins/gacha/engine/test/card_simulator.py:6]:
```
from .balanced_cards import *
from .dmg_cards import *
from .hp_cards import *
from .meme_cards import *
```
I guess the way we import in `__init__.py` preserves the individual modules?

**Pull request merged**